docs/introduction.md

```diff
@@ -1,19 +1,3 @@
 ---
 sidebar_position: 1
----
-
-# Introduction
-
-## Frontend
-
-## Backend
-
-## Database
-
-## Cloud
-
-## Testing
-
-## Architecture
-
-## Legacy code
+---
```

Comment:
The document general-rules-and-definitions.md also has position 1.
It looks to me like the introduction should be first.

Maybe: 
```
sidebar_position: 0
```